include/exiv2/tiffimage.hpp

```diff
@@ -90,7 +90,7 @@ namespace Exiv2 {
                 not valid (does not look like data of the specific image type).
           @warning This function is not thread safe and intended for exiv2 -p{S|R} as a file debugging aid
          */
-        virtual void printStructure(std::ostream& out, PrintStructureOption option,int depth=-1);
```

Comment:
The previous version sure looks odd. @clanmills Do you know why `depth` defaults to -1? You have added that parameter in fd5e983746c336336039e91cb6b656cf8eeccdea and set a default in the base class and here, so I suspect there is a reason for that.